pkg/meta/base.go

```diff
@@ -1363,9 +1365,8 @@ func (m *baseMeta) cleanupTrash() {
 	}
 }
 
-func (m *baseMeta) doCleanupTrash(force bool) {
+func (m *baseMeta) CleanupTrashBefore(ctx Context, edge time.Time, increProgress func()) {
```

Comment:
just export `CleanupTrash` with a limit.

Reply:
Now the `doCleanupTrash` is based on `CleanupTrashBefore`.

Reply:
```go
func (m *baseMeta) doCleanupTrash(force bool) {
	edge := time.Now().Add(-time.Duration(24*m.fmt.TrashDays+1) * time.Hour)
	if force {
		edge = time.Now()
	}
	m.CleanupTrashBefore(Background, edge, nil)
}
```